pcdsdevices/sim/daq.py

```diff
@@ -118,14 +118,21 @@ def _pick_duration(self, events, l1t_events, l3t_events, duration):
                 else:
                     return ev / 120
         if duration is not None:
-            if isinstance(duration, list):
+            if not isinstance(duration, list):
```

Comment:
Before you couldn't have a list and now you need a list? Not doubting you, just want to know why.

Reply:
I originally observed pydaq accepting integer arguments and rejecting [seconds, nanoseconds]. This is because for some integer values this occasionally works, and that for very small values of [seconds, nanoseconds] the daq will spazz out. I originally misattributed the spazzing of the daq to the list argument rather than the magnitude of the list argument and completely missed the fact that duration=int only worked for a small number of cases.

Reply:
Interesting.